templates/master/00-master/sno/files/sno-forcedns-fix.yaml

```diff
@@ -0,0 +1,13 @@
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/forcedns-rhel9-fix"
+contents:
+  inline: |
+    #!/bin/bash
+    # this script was added as upgrade fix for assisted sno installation were forcedns dispatcher script misses shebang
+    filename="/etc/NetworkManager/dispatcher.d/forcedns"
+    if [ ! -f "$filename" ];
+    then
+      exit
```

Comment:
force an exit status to be sure that the script returns what is supposed to return

```
exit 0 # Script finishes OK
```
or 
```
exit 1 # Script finishes with error
```